sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -3258,6 +3258,10 @@ class ModelOperationInfo(object):
     :vartype kind: str
     :ivar resource_location: URL of the resource targeted by this operation.
     :vartype resource_location: str
+    :ivar api_version: API version used to create this operation.
+    :vartype api_version: str
+    :ivar tags: List of key-value tag attributes associated with the model.
```

Comment:
should we say "user-defined" kv attributes here?

Reply:
That seems pretty safe, I did pause for a moment in case this set of tags could ever include non-user-defined tags, but that is probably very unlikely. I'll add this and just confirm with the service team if this is suitable wording for this property.